### PR TITLE
SCT-6 added oauth functionality for zoom

### PR DESCRIPTION
**Pull request by katiemcgoff**
Don't be alarmed, there are really only two files that are important here - the zoom views and the zoom services files. The services file contains the main zoom oauth integration, and the views just wrap up that functionality for the two URLs. Everything else is architecture (urls, new org model fields) and a LOT of tests.

### CODE REVIEW

File: api/models/organization.py [hunk 1]
```diff
@@ -6,16 +6,19 @@ class Organization(models.Model):
     name = models.CharField(max_length=250)
     timezone = models.CharField(
         "Organization Timezone", choices=[(x, x) for x in pytz.all_timezones], max_length=200, blank=True, null=True)
+    disabled = models.BooleanField(default=False)
```
Comment (katiemcgoff) [inline api/models/organization.py:9]: unrelated change - I just moved this up to group the slack and zoom fields together.

**Review by DawsonDease — APPROVED** [review]

**Review by JaycobDuffel — APPROVED** [review]
